README.md

```diff
@@ -196,6 +196,7 @@ A `*` denotes required configuration.
 | `source_path`          | List of directory paths | Directories to search for source files. Note: source_path for dependencies downloaded using `maven_downloads` configuration is added automatically without the need to specify here. |
 | `class_path`           | List of directory / JAR paths | Classpath for API summary generation. This should include any JAR dependencies of the source files in `source_path`. |
 | `classes` *            | List of qualified class / package names | List of qualified class / package names. `source_path` will be scanned assuming the sources follow standard java-ish hierarchy. That is a.b.c either maps to a directory `a/b/c` or a class file `a/b/c.java`.  |
+| `suspend_fun_to_async` | True/False | Converting Kotlin's suspend functions to Dart's async functions. Defaults to False.  |
```

Comment:
Should this default to true instead?

Would users ever _not_ want this?

Reply:
This was a decision a made a while back because we currently understand if a function is `suspend fun` when the last parameter passed to that function is `kotlin.coroutines.Continuation`, this is of course not always correct. We could say that it's correct 99% of the time and default it to True. Or we can default it to True when we have a better approach of detecting `suspend fun`ness!